2-mandatory/3-financial-times.js

```diff
@@ -1,81 +1,113 @@
+const ARTICLE_TITLES = [
+    "Streaming wars drive media groups to spend more than $100bn on new content",
+    "Amazon Prime Video India country head: streaming is driving a TV revolution",
+    "Aerospace chiefs prepare for bumpy ride in recovery of long-haul flights",
+    "British companies look to muscle in on US retail investing boom",
+    "Libor to take firm step towards oblivion on New Year's Day",
+    "Audit profession unattractive to new recruits, says PwC boss",
+    "Chinese social media users blast Elon Musk over near miss in space",
+    "Companies raise over $12tn in 'blockbuster' year for global capital markets",
+    "The three questions that dominate investment",
+    "Brussels urges Chile's incoming president to endorse EU trade deal",
+];
 /*
     Imagine you are working on the Financial Times web site! They have a list of article titles stored in an array.
 
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
+
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    let arrLingth65 = allArticleTitles.filter(
+        (item) => item.split("").length <= 65
+    );
+    return arrLingth65;
 }
+// console.log(potentialHeadlines(ARTICLE_TITLES));
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let arrLingth65 = allArticleTitles.filter(
+        (item) => item.split("").length <= 65
+    );
+    return arrLingth65[arrLingth65.length - 1];
```

Comment:
This logic returns the last headline which has 65 characters or less - by co-incidence this is also the headline with the fewest words so your test passes.. But if the order of ARTCLE_TITLES is changed, the test will fail. If you have time, try to re-implement this with the correct logic.